### PR TITLE
small refinements in lab02 exercise

### PR DESCRIPTION
**Pull request by kretes**

### CODE REVIEW

File: labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala [hunk 1]
```diff
@@ -40,7 +40,7 @@ object CollectionExercise01 {
 
 object CollectionExercise02 {
 
-  class Person(val age: Int, val name: String)
+  case class Person(val age: Int, val name: String)
```
Comment (raboof) [inline labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala]: At this point in the exercise we haven't covered case classes yet, so we'd prefer to postpone that to later.

File: labs/src/test/scala/org/scalalabs/basic/lab02/CollectionExerciseTest.scala [hunk 2]
```diff
@@ -37,7 +37,7 @@ class CollectionExerciseTest extends Specification {
       val erik = new Person(24, "Erik")
       val susy = new Person(40, "Susy")
 
-      val result = CollectionExercise02.groupAdultsPerAgeGroup(Seq(jack, duke, jeniffer, erik, susy))
+      val result = CollectionExercise02.groupAdultsPerAgeGroup(Seq(jack, jeniffer, duke, erik, susy))
```
Comment (raboof) [inline labs/src/test/scala/org/scalalabs/basic/lab02/CollectionExerciseTest.scala]: What is this change intended to solve?

Comment (kretes) [inline labs/src/test/scala/org/scalalabs/basic/lab02/CollectionExerciseTest.scala]: a long time since I did that, but I suppose that to test that one does a 'sort' we should provide a different order as an input.
otherwise an implementation could ignore sorting and the test would pass. this enforces implementation to really do sorting here

File: labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala (hunk 1 above)
Comment (kretes) [inline labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala]: ok
